src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -207,6 +207,17 @@ function WebGLShadowMap( _renderer, _objects, maxTextureSize ) {
 
 				_lookTarget.setFromMatrixPosition( light.target.matrixWorld );
 				shadowCamera.lookAt( _lookTarget );
+
+				// FIXME: feels wrong here, but the only other solution I
+				//   can come up with would be to have updating the shadow-cameras
+				//   fully managed by the *LightShadow classes.
+
+				if ( shadow.isProjectorShadow ) {
+
+					shadowCamera.rotateZ( light.rotation.z );
```

Comment:
I'm not sure this property adds much. Consider removing it?

Reply:
What I want to achieve is full 3DOF rotation of the projector. The first two are already handled by the `lookAt(target)` I copied from the other light-types, so there needs to be a way to rotate the projector relative to it's up-direction. If this would be removed, the only option to do that would be to modify the up-direction which feels a bit weird.

One refactoring I would love to do is to make the target optional for all light-types: If no target is specified, the default `Object3D.rotation` property would be used instead. That would allow me to implement this rotation in client-code if required.